setup.py

```diff
@@ -44,7 +44,12 @@ def parse(root, config):
         try:
             return parse_pkginfo(root, config)
         except OSError:
-            return git.parse(root, config=config)
+            version = git.parse(root, config=config)
+            if version is not None:
+                return version
+            from setuptools_scm import hg
```

Comment:
lets move the import up and return `git.parse(...) or hg.parse(...)`

Reply:
Oh yes, much nicer!